yew/src/virtual_dom/vtag.rs

```diff
@@ -448,18 +442,25 @@ impl VDiff for VTag {
         }
 
         // Process children
-        self.children
-            .apply(scope, &element, None, ancestor.map(|a| a.children.into()));
-
-        let node = self.reference.as_ref().map(|e| {
-            let node = cfg_match! {
-                feature = "std_web" => e.as_node(),
-                feature = "web_sys" => e.deref(),
-            };
-            node.to_owned()
-        });
-        self.node_ref.set(node.clone());
-        node
+        self.children.apply(
+            parent_scope,
+            &element,
+            VDiffNodePosition::FirstChild,
```

Comment:
I think that `LastChild` is synonymous with `FirstChild` here. Could we replace it and remove `FirstChild` altogether?